README.md

```diff
@@ -1,2 +1,19 @@
 # MockHospitalData
-This repo contains mock data to use with this project.
+
+This repo contains mock data apis to use with this project.
+
+## Prerequisites
+
+``` bash
+pip install -r ./requirements.txt
+```
+
```

Comment:
Need another descrepition of the  simulated hospital-information-system's format ( xlsx / REST api / access to DB )